galaxy_ng/app/constants.py

```diff
@@ -30,6 +30,12 @@ class DeploymentMode(enum.Enum):
         # Category to group the permission in the UI.
         "ui_category": _("Collection Namespaces"),
     },
+    "galaxy.view_namespace": {
```

Comment:
If you add a new permission here, would you need to add it to existing roles like `galaxy.collection_namespace_owner` which currently offers `galaxy.change_namespace`? Would some other API access logic need to be rewritten to utilize the new permission?

Reply:
@AlanCoding @jctanner This file should contain the set of permissions that we actually care about for enforcement. The problem is that pulp/django ship with a huge pile of random permissions, most of which we don't care about. `galaxy.view_namespaces` is a good example. Namespaces are always readable by everyone who is authenticated. We don't want to expose that permission to the UI or to users because adding it to a role does nothing.

@AlanCoding this is why I was so against using django permissions for DAB RBAC. It's practically impossible to get a full list of permissions that are actually enforced and actually do something in the system.

Reply:
This led to me file https://github.com/ansible/django-ansible-base/issues/457. But it should be super minor, I want to declare a change is probably necessary in DAB RBAC for the documented reasons, but I'm completely confident that won't be challenging to do. We've just haven't done it yet.